crates/bevy_pbr/src/meshlet/mod.rs

```diff
@@ -169,6 +178,15 @@ impl Plugin for MeshletPlugin {
             return;
         };
 
+        if !render_app
+            .world()
+            .resource::<RenderDevice>()
+            .features()
+            .contains(WgpuFeatures::PUSH_CONSTANTS)
+        {
+            return;
```

Comment:
Not sure what the exact message should be but I think users should be warned. I know it's experimental but if we ever have a tester that doesn't know this it will make it easier to debug.
```suggestion
            warn!("Your device doesn't support push constants which is needed for meshlet rendering")
            return;
```

Reply:
Yeah, that's fair.